exporter/elasticsearchexporter/exporter.go

```diff
@@ -132,9 +200,9 @@ func newElasticsearchClient(logger *zap.Logger, config *Config) (*esClientCurren
 
 		// configure retry behavior
 		RetryOnStatus:        retryOnStatus,
-		DisableRetry:         false,
-		EnableRetryOnTimeout: true,
-		MaxRetries:           config.Retry.MaxRequests,
+		DisableRetry:         !config.Retry.Enabled,
+		EnableRetryOnTimeout: !config.Retry.Enabled,
+		MaxRetries:           maxAttempts,
```

Comment:
Naming clarity: I generally think of "attempts" and "retries" as being off by 1 -- i.e. if the max "attempts" is 3, that means you only "retry" (try again) twice. This convention isn't universal, but it seems confusing to use both terms in close proximity. It would be nice to stick with the same term in both structs and local variables if possible, or to note the discrepancy at the definition sites if not.

Reply:
The structs is not defined in the exporter but the go-elasticsearch dependency. Debugging the tests I learned that the library starts the count with 1 (if 0 is passed the library will default to 3 retries instead of disabling retries). Will add a clarifying comment to `maxAttempts`. After testing I figured that the behavior is more similar to an attempt, which is what I named the variable with.